shared/overview/2.x.md

```diff
@@ -90,7 +90,7 @@ A diagram of our read-only rootfs can be seen below:
 
 ## Image Partition Layout
 
-The first partition, `resin-boot`, host boot important bits according to each board (e.g. kernel image, bootloader image). It also holds a very important file that you will find mentioned elsewhere in this document (i.e. `config.json`). The config.json file is the central point of configuring {{ $names.os.lower }} and defining its behaviour, for example you can set your hostname, allow persistent logging, etc.
+The first partition, `resin-boot`, host boot important bits according to each board (e.g. kernel image, bootloader image). It also holds a very important file that you will find mentioned elsewhere in this document (i.e. `config.json`). The `config.json` file is the central point of configuring {{ $names.os.lower }} and defining its behaviour, for example you can set your hostname, allow persistent logging, etc.
```

Comment:
Maybe "host boot important bits" -> "holds important boot files" ?
At the very least, "host" should be changed to either "hosts" or "holds".

Reply:
thanks for the note @lurch, opened https://github.com/balena-io/docs/pull/1115 to address